code/index.html

```diff
@@ -3,28 +3,198 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <title>Pawsitive talk</title>
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Patrick+Hand&display=swap" rel="stylesheet">
+    <link rel="icon" type="image/png" href="./dog-paw.png"/>
```

Comment:
Nice detail, maybe you'd like to add an image folder to structure your project a bit 🌟